src/Hooks/useSortedTokensByFiatValue/useSortedTokensByFiatValue.ts

```diff
@@ -112,7 +114,34 @@ export const useSortedTokensByFiatValue = (accountAddress?: string) => {
         accountAddress: address,
     })
 
+    const hasTokensWithBalance = tokenBalances.some(token => !BigNutils(token.balance.balance).isZero)
+    const isNewUserWithNoTokens = !hasAnyVeBetterActions && !hasTokensWithBalance
```

Comment:
Add a `useMemo` here, otherwise `sortedTokens` will be computer way more often than it should